src/System.Data.SqlClient/tests/ManualTests/SQL/ParameterTest/SqlAdapterUpdateBatch.cs

```diff
@@ -9,13 +9,10 @@ namespace System.Data.SqlClient.ManualTesting.Tests
 {
     public class SqlAdapterUpdateBatch
     {
-        private static string tcpConnStr = DataTestUtility.TcpConnStr;
-        private static string tableName = "BatchDemoTable";
-        private static List<EventInfo> entities;
-
         [Fact]
```

Comment:
Should this be [CheckConnStrSetupFact] ? The test relies on a server being configured and CI doesn't have the SQL Server configured.

Reply:
Thanks. Updated the test.